flink-libraries/flink-table/src/main/scala/org/apache/flink/api/table/table.scala

```diff
@@ -446,9 +446,9 @@ class Table(
   }
 
   /**
-    * Intersect two [[Table]]s with duplicate records removed. Intersect returns rows only
-    * from the left table that are identical to a row in the right table.
-    * Similar to an SQL INTERSECT. The fields of the two union operations must fully overlap.
+    * Intersect two [[Table]]s with duplicate records removed. Intersect returns records that
+    * exist in both tables, and emit exactly once. Similar to an SQL INTERSECT. The fields of
```

Comment:
I would rephrase this to "Intersect returns records that exist in both tables. If a record is present one or both tables more than once, it is returned just once, i.e., the resulting table has no duplicate records."